web/app/views/bloggs/index.html.erb

```diff
@@ -2,6 +2,21 @@
 
 <%= link_to 'New article', new_blogg_path %>
 
+
+<% if flash[:error] %>
+    <p class="notice"><%= flash[:error] %></p>
+<% end %>
+
+<% if user_signed_in? %>
+    <li>
+      <%= link_to('Logout', destroy_user_session_path, :method => :delete) %>
```

Comment:
:method is a keyword that is talking about an HTTP Method.  AKA GET, POST, DELETE.  So in this case we want to make a DELETE request to the destroy_user_session_path (Named Routing will map this value to localhost:3000/blogg/3/delete or something like that),